### ISSUE TITLE
ENOENT: no such file or directory, open '.mtl'

### ISSUE THREAD

**Issue by vitalspace** [battlelinegames/obj2asc#1]
Hi I am trying to use the cli tool but it throws me the following errors.

![Image of error](https://i.imgur.com/0PJjrIN.png)

as you can see I have the ".mtl" file in the same folder where the ".obj" file is located.

![Image of error](https://i.imgur.com/XEmAql2.png)

**Comment by battlelinegames** [comment]
Can you open up casa.obj and look for the mtllib line?  There isn't any directory information in the .obj file by any chance, is there?

Also, I've only tested this with Blender.  Do you happen to be using a different app for creating your obj file?

Thanks!

**Comment by vitalspace** [comment]
I did the example with sketchup it is a simple house.

here is the line

![Image of line](https://i.imgur.com/IQRUfRy.png)

**Comment by battlelinegames** [comment]
Would you mind sending me those two files on Discord?  I'd like to try it and see what happens.  From what I see it looks like it should work, but the only files I tested this on came from Blender, so maybe there is something I'm missing.

**Comment by battlelinegames** [comment]
Carriage returns in that file appear to be throwing it off
I try to strip out all the CRs earlier in the code, but that's not working for some reason.
If I remove it when I try to open the file it seems to work

**Comment by battlelinegames** [comment]
After I fixed the CR problem, there are several other issues that must be a difference between the .obj generated by Blender vs. Sketchup.  There is not an active object declared by Sketchup, so I added this line:

```
o casa
```

After that, the material file did not contain 'Ke' values.

The next problem I hit was that the faces were not triangulated.  Is there an option to triangulate the faces when you export to .obj from Sketchup?
